packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -204,12 +204,26 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         // // Check gas right before the call to get total gas consumed by OVM transaction.
         // uint256 gasProvided = gasleft();
 
-        // Run the transaction, make sure to meter the gas usage.
-        (, bytes memory returndata) = ovmCALL(
-            _transaction.gasLimit - gasMeterConfig.minTransactionGasLimit,
-            _transaction.entrypoint,
-            _transaction.data
-        );
+        bytes memory returndata;
```

Comment:
If we move this encoding into the `L2ChugsplashDeployer`, similarly to how the `OVM_ECDSAContractAccount`'s return type works, then I think this diff can instead just be:

```
address entrypoint = _isUpgrading() ? 0x420000000000000000000000000000000000000D : _transaction.entrypoint;
(, bytes memory returndata) = ovmCALL(
            _transaction.gasLimit - gasMeterConfig.minTransactionGasLimit,
            entrypoint,
            _transaction.data
        );
```

Reply:
Maybe this makes it a little harder to test though

Reply:
Yeah harder to test, unfortunately also means that we'd have to handle reverts in a weird way. I think this is better for now.